### PR TITLE
Bug/gcloud fix

### PR DESCRIPTION
**Pull request by sgehrman**

### CODE REVIEW

File: server/port.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
-const argv = require('./argv');
+// const argv = require('./argv');
 
-module.exports = parseInt(argv.port || process.env.PORT || '3000', 10);
+// module.exports = parseInt(argv.port || process.env.PORT || '3000', 10);
+module.exports = parseInt('8080', 10);
```
Comment (traylewin) [inline server/port.js]: Please don't change this code if we don't have to. Looks like we just need to have an .env file with PORT=8080 in the root of the project to make this work. Add a .env file to the repo

File: package.json [hunk 2]
```diff
@@ -30,12 +26,12 @@
     "preinstall": "npm run npmcheckversion",
     "postinstall": "npm run build:dll",
     "prebuild": "npm run build:clean",
-    "build": "cross-env NODE_ENV=production webpack --config internals/webpack/webpack.prod.babel.js --color -p --progress --hide-modules --display-optimization-bailout",
+    "build": "cross-env NODE_ENV=production webpack --config internals/webpack/webpack.dev.babel.js --color -p --progress --hide-modules --display-optimization-bailout",
```
Comment (traylewin) [inline package.json:29]: What is triggering the build process? Do we have to do it manually? If so, update the repo readme to include this step.

**Review by traylewin — CHANGES_REQUESTED** [review]
Please make a few changes

File: server/port.js (hunk 1 above)
Comment (sgehrman) [reply server/port.js]: I thought about adding the .env, but that would require everyone remembering to do that.  So, I guess I would add that to the readme too?  Or check that into github?

**Review by traylewin — APPROVED** [review]

**Pull request merged**